runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -182,6 +197,24 @@ public void onChatMessage(ChatMessage event)
 			{
 				notifier.notify("Your Ring of Recoil has shattered");
 			}
+			else if (ringOfForgingCheckMatcher.find())
+			{
+				updateRingOfForgingCharges(Integer.parseInt(ringOfForgingCheckMatcher.group(1)));
```

Comment:
this will error when the captured group is `one`. there should be a test for this too.